packages/dynamodb/package.json

```diff
@@ -19,7 +19,9 @@
     "access": "public"
   },
   "scripts": {
-    "test": "echo \"Fix tests.\";exit 0",
+    "test:default": "jest",
+    "test:custom": "CUSTOM_MODEL=1 jest",
```

Comment:
These tests are failing.

Reply:
I just updated the PR, all tests should pass now. There was quite a few changes, overall the code is simpler and has less typescript added complexity.